eslint.config.mjs

```diff
@@ -65,6 +65,7 @@ export default tsEslint.config({
     "sonarjs/no-element-overwrite": "error",
     "sonarjs/no-identical-conditions": "error",
     "sonarjs/no-identical-expressions": "error",
+    "sonarjs/new-cap": "off",
```

Comment:
Easier I show you in a screenshot

It's complaining because the syntax goes `Capital.Capital` it appears, all of the typebox stuff is throwing this error because all of their fns and types follow that pattern.

![image](https://github.com/user-attachments/assets/92d48768-3e6b-4425-b244-d9f84f54675b)